### DISCUSSION TITLE
When trying to list/edit data, I get the message "Something went wrong while processing your request."

### DISCUSSION THREAD

**Discussion by renatocaliari** [pockethost/pockethost#259] [Q&A]
When trying to list/edit data of a record in some tables (including "users"), I get the error "Something went wrong while processing your request" and I don't know how to proceed. 

The "Request Logs" doesn't show any errors.

![Screenshot 2023-07-24 06 49 31](https://github.com/benallfree/pockethost/assets/1719651/ac69679f-8217-4138-b604-8487c32b8458)

**Comment by Savit3r** [comment]
Pockethost seems to be not working with everyone at the moment...

**Reply by renatocaliari** [reply]
Thank you for notifying your situation as well.
At this moment, it seems that Pockethost has entered a loop, continuously displaying this message. 

I tried accessing the home page of the website (https://pockethost.io/), but my instances no longer appear for me to log in.

**Comment by benallfree** [comment]
Moving to #223
